### ISSUE TITLE
Hide ocean in some locations

### ISSUE THREAD

**Issue by DeividAndreola** [UE4-OceanProject/OceanProject#56]
Is there a way to hide the ocean in some areas? Ex: My city is surrounded by the ocean, but some areas of the city, such a subway is below the sea level.

**Comment by TheZoc** [comment]
I think you might need a custom solution for that, but If you find a way, I'd love if you could share with as a Pull Request!

**Comment by robinhood90** [comment]
this example
https://i.hizliresim.com/kXVX2A.jpg

**Comment by TheZoc** [comment]
@hilaladam Do you think you could make a sample/example blueprint and submit to us as a PR? :)

**Comment by tdw46** [comment]
@TheZoc I have experimented with @hilaladam 's method to no avail. Seems like an interesting starting point if you want to hide the ocean only when an actor overlaps in certain locations. However, if you are looking to selectively hide the ocean, I would suggest this approach in the Ocean Materials:

https://www.youtube.com/watch?v=XYPPd5oLoPM&index=17&list=PL3kjeDjmmGQiKWtdTssG1hhVNcUzx4Aqd&t=0s

I have experimented with the above approach and it works for every material I have attempted it on so far except "M_Ocean_Ultra" "M_Ocean" etc. It seems the opacity node in the material isn't the only thing determining whether the final BP_Ocean renders opaque or not?
